cmd/juju/commands/main.go

```diff
@@ -63,13 +63,51 @@ func init() {
 //  Main
 
 var jujuDoc = `
-Juju provides easy, intelligent application orchestration on top of cloud
-infrastructure providers such as Amazon EC2, MaaS, OpenStack, Windows, Azure,
-or your local machine.
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
+cloud infrastructure providers such as Amazon, Google, Microsoft, Openstack,
+MAAS (and more!), or even on your local machine via LXD.
 
-See https://discourse.jujucharms.com/ for ideas, documentation and FAQ.
+See https://juju.is for getting started tutorials and additional documentation.
 `
 
+var usageHelp = `
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
+cloud infrastructure providers such as Amazon, Google, Microsoft, Openstack,
+MAAS (and more!), or even your local machine via LXD.
+
+See https://juju.is for getting started tutorials and additional documentation.
+
+Starter commands:
+
+    add-k8s             Adds a k8s endpoint and credential to Juju.
+    bootstrap           Initializes a cloud environment.
+    add-model           Adds a hosted model.
+    deploy              Deploys a new application.
+    status              Displays the current status of Juju, applications, and units.
+    add-unit            Adds extra units of a deployed application.
+    relate              Adds a relation between two applications.
+    expose              Makes an application publicly available over the network.
+    models              Lists models a user can access on a controller.
+    controllers         Lists all controllers.
+    whoami              Display the current controller, model and logged in user name. 
+    switch              Selects or identifies the current controller and model.
+    add-cloud           Adds a user-defined cloud to Juju.
+    add-credential      Adds or replaces credentials for a cloud.
+
+REPL mode:
```

Comment:
REPL seems very tech focused. I would probably call this "Interactive mode:"

Reply:
The term repl makes me happy. It's so cheerful. But I agree that we should probably call it "interactive" or "shell" mode.

Reply:
Changed to "Interactive Mode"